pip/pip-272.md

```diff
@@ -0,0 +1,214 @@
+<!--
```

Comment:
Would it be possible to remove all the comments? There are just guidelines while you write. No need for a final PIP.

Reply:
sure, I will remove them